help/develop/admin/deploy/cloudformation/cloudformation.yml

```diff
@@ -813,7 +936,7 @@ Resources:
         - ContainerPort: 5005
           HostPort: 5005
           Protocol: tcp
-      Cpu: '1024'
+      Cpu: '512'
```

Comment:
Move to the container definition and match the values from *_container_cpu variables in terraform defaults

Reply:
And leave it ad it was for FARGATE

Reply:
```suggestion
      Cpu: '1024'
```